.gitignore

```diff
@@ -4,4 +4,7 @@ Cargo.lock
 *.sublime-project
 .idea/
 
+.idea
```

Comment:
There's already an entry on .gitignore to ignore `.idea/`, was it not working for you?